src/pages/index.astro

```diff
@@ -49,47 +48,47 @@ const meta = {
+          title: 'E-handelsstrategi og rådgivning:',
           description:
-            'Widgets made with Tailwind CSS ready to be used in Marketing Websites, SaaS, Blogs, Personal Profiles, Small Business...',
+            'Vårt ekspertteam gir deg omfattende e-handelsstrategi og rådgivning for å hjelpe deg med å maksimere salg og vekst. Vi analyserer din målgruppe, konkurrenter og markedssegmenter for å utvikle skreddersydde strategier som passer til din unike virksomhet.',
           icon: 'tabler:components',
         },
         {
-          title: 'Best Practices',
+          title: 'Integrerte betalings- og fraktløsninger',
           description:
-            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi sagittis, quam nec venenatis lobortis, mi risus tempus nulla.',
+            'Vi hjelper deg med å implementere og optimalisere integrerte betalings- og fraktløsninger som gjør det enkelt for kundene å fullføre kjøpene. Vi sikrer at sikkerheten og brukervennligheten er i toppklasse, slik at dine kunder kan handle trygt og smidig.',
           icon: 'tabler:list-check',
         },
       ],
       [
         {
-          title: 'Excellent Page Speed',
+          title: 'Optimalisert brukeropplevelse',
           description:
-            'Having a good page speed impacts organic search ranking, improves user experience (UI/UX) and increase conversion rates.',
+            'Vi fokuserer på å skape en sømløs brukeropplevelse som øker konverteringer og engasjement. Fra responsivt design til intuitiv navigasjon og rask lasting, sørger vi for at kundene dine får en enkel og behagelig opplevelse når de besøker nettbutikken din.',
           icon: 'tabler:rocket',
         },
         {
-          title: 'Search Engine Optimization (SEO)',
+          title: 'Optimalisert for synlighet(SEO)',
           description:
-            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi sagittis, quam nec venenatis lobortis, mi risus tempus nulla.',
+            'Med Okane får du en nettbutikk som er nøye optimalisert for søkemotorer. Vi implementerer beste praksis for SEO, inkludert riktig bruk av søkeord, optimalisering av metadata, optimal sidestruktur og mobilvennlighet. Dette hjelper deg med å øke synligheten din i søkemotorresultatene, slik at potensielle kunder lettere finner deg og øker sjansene for organisk trafikk og salg.',
           icon: 'tabler:arrows-right-left',
         },
         {
-          title: 'Open to new ideas and contributions',
+          title: 'Kontinuerlig støtte og oppdateringer',
           description:
-            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi sagittis, quam nec venenatis lobortis, mi risus tempus nulla.',
+            'Vi er her for deg også etter at nettbutikken er lansert. Vårt dedikerte supportteam gir kontinuerlig støtte og oppdateringer for å sikre at nettbutikken din fungerer problemfritt og holder tritt med de siste trender og krav.',
```

Comment:
"Vårt dedikerte supportteam gir kontinuerlig støtte og oppdateringer" sounds like we are providing them free support of the webstore.

"Vårt dedikerte supportteam kan gi kontinuerlig støtte og oppdateringer" is better?